backtrader/stores/ibstore.py

```diff
@@ -843,7 +845,9 @@ def reqMktData(self, contract):
         if contract.m_secType in ['CASH', 'CFD']:
             self.iscash[tickerId] = True
             ticks = ''  # cash markets do not get RTVOLUME
-
+            if what is 'ASK':
```

Comment:
This will need a change. Using "is" for a comparison with a string is bound to break